data_steward/analytics/cdr_ops/rdr_export_qc.py

```diff
@@ -642,4 +643,29 @@
 group by value_source_concept_id, value_as_concept_id
 """)
 query = tpl.render(new_rdr=new_rdr, project_id=project_id)
-execute(client, query)
\ No newline at end of file
+execute(client, query)
+
+# # Verify that no joint Personal Family Medical History survey information exists in the Dataset.
+# The survey launched on 2021/11/01.  This date has been used to help identify this data.
+# Checking PFMH concept_codes in conjunction with the survey date.
+
+tpl = JINJA_ENV.from_string("""
+SELECT
+  observation_source_value,
+  COUNT(*) AS n_rows_violation
+FROM
+  `{{project_id}}.{{dataset}}.observation`
+WHERE
+  -- separated surveys were removed from participant portal on 2021-11-01 --
+  -- combined survey was launced on 2021-11-01 --
+  observation_date > '2021-10-31'
+  AND LOWER(observation_source_value) IN ({{combined_pfmhh_concepts}})
```

Comment:
Should this also check in `observation_source_concept_id`?

Reply:
we only have a list of concept codes that we are using in the cleaning rule as well, and this is potentially a one-time thing. we are going to remove this step by nov CDR.

Reply:
Ah, okay. This is fine then.